src/sonic-config-engine/tests/test_minigraph_case.py

```diff
@@ -79,11 +79,10 @@ def test_minigraph_interfaces(self):
         self.assertEqual(output.strip(), "[('Ethernet0', '10.0.0.58/31'), 'Ethernet0', ('Ethernet0', 'FC00::75/126')]")
 
     def test_minigraph_vlans(self):
-        argument = '-m "' + self.sample_graph + '" -p "' + self.port_config + '" -v VLAN'
-        output = self.run_script(argument)
+        vlan_table = minigraph.parse_xml(self.sample_graph, port_config_file=self.port_config)["VLAN"]
```

Comment:
Hmm, with sonic-cfggen, we are testing more layers. I think the name might be confusing as one might think this is a unit test for minigraph module. I'd prefer to use the sonic-cfggen command for the reason mentioned before and maybe we can unit test for specific modules.

Reply:
Good point, I'll put the `sonic-cfggen` call back. Thanks!